addons/sms/models/mail_thread.py

```diff
@@ -146,6 +147,13 @@ def _sms_get_recipients_info(self, force_field=False, partner_fallback=True):
                 }
         return result
 
+    @api.returns('mail.message', lambda value: value.id)
+    def message_post(self, *args, body='', message_type='notification', **kwargs):
```

Comment:
Would add a comment in code explaining why we do that :)  (in addition to commit messages, always a good idea to have at least an hint when reading the code)